cocos/rendering/custom/types.ts

```diff
@@ -475,6 +475,7 @@ export class MovePair {
         targetMostDetailedMip = 0,
         targetFirstSlice = 0,
         targetPlaneSlice = 0,
+        possibleUsage = AccessFlagBit.NONE,
```

Comment:
How is this `possibleUsage` used? @hana-alice

Reply:
1. If resA will be used in the render graph later, I think possibleUsage is not helpful? Since the usage can be deduced from the following usages.
2. If move is the last operation of the resA, then possibleUsage is useful.

Reply:
Yes, only the last access of moved resource in a frame will have a check on possibleUsage. It's sort of hint and ignored if there is a coming access.

Reply:
I see.